src/lib/components/chat/ChatWindow.svelte

```diff
@@ -226,12 +246,19 @@
 				</p>
 				{#if messages.length}
 					<button
-						class="flex flex-none items-center hover:text-gray-400 hover:underline max-sm:rounded-lg max-sm:bg-gray-50 max-sm:px-2.5 dark:max-sm:bg-gray-800"
+						class="flex flex-none items-center hover:text-gray-400 max-sm:rounded-lg max-sm:bg-gray-50 max-sm:px-2.5 dark:max-sm:bg-gray-800"
 						type="button"
+						class:hover:underline={!$isConversationShared}
 						on:click={() => dispatch("share")}
```

Comment:
```suggestion
						on:click={() => {
							dispatch("share");
							isSharedRecently = true;
							setTimeout(() => {
								isSharedRecently = false;
							}, 2000);
						}}
```

Reply:
this is the main piece, we dispatch the event and set the timeout here, shouldn't need much else

Reply:
handled in https://github.com/huggingface/chat-ui/pull/645/commits/fdbb4cafbf846c95557cb7e2fe0ce45effcd4c76